QuantLib/ql/termstructures/volatility/optionlet/optionletstripper1.cpp

```diff
@@ -24,6 +24,7 @@
 #include <ql/termstructures/volatility/optionlet/optionletstripper1.hpp>
 #include <ql/instruments/makecapfloor.hpp>
 #include <ql/pricingengines/capfloor/blackcapfloorengine.hpp>
+#include <ql/pricingengine.hpp>
```

Comment:
Probably not. One could argue,  that it is"good style" to always directly include the headers referred to rather then indirectly using them. What is current QL best practice?